taxi/forms.py

```diff
@@ -0,0 +1,49 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.validators import RegexValidator
+
+from taxi.models import Car
+
+
+class LicenseNumberMixin(forms.ModelForm):
+    license_number = forms.CharField(
+        required=True,
+        max_length=8,
+        validators=[
+            RegexValidator(
+                regex=r"^[A-Z]{3}[0-9]{5}$",
+                message=(
+                    "License number must be 3 uppercase letters and 5 digits."
+                )
+            )
+        ]
+    )
+
+
+class DriverCreationForm(LicenseNumberMixin, UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = get_user_model()
+        fields = UserCreationForm.Meta.fields + (
+            "first_name",
+            "last_name",
+            "license_number",
+        )
+
+
+class DriverLicenseUpdateForm(LicenseNumberMixin, forms.ModelForm):

```

Comment:
using inheritance here would be a better approach (add mixin is also possible, but in this case UpdateForm is exactly the same as CreateForm and only fields in Meta are different)